portfolio/src/main/webapp/styles/style.css

```diff
@@ -1,35 +1,36 @@
 /* General Elements */
 body {
-  color:rgb(160, 160, 160);
-  background-color: #02003C;
+  color:white;
+  background-color: #41485B;
 }
 
 img {
   height: 10%;
   width: 10%;
 }
 
+h1 {
+  font-family: 'Roboto', sans-serif;
+}
+
 .content {
+  font-family: 'Roboto Condensed', sans-serif;
   padding-top: 2vh;
```

Comment:
what's the difference between vh and vw?

Reply:
1 vw is 1% of the width of the viewport where vh is for viewport height.